dom.bs

```diff
@@ -8679,6 +8679,18 @@ interface DOMTokenList {
 <p>A {{DOMTokenList}} object also has an associated <a for="/">element</a> and an <a>attribute</a>'s
 <a for=Attr>local name</a>.
 
+<a lt="Other applicable specifications">Specifications</a> may define
+<dfn export for=Node id=concept-supported-keywords>supported keywords</dfn> for a {{DOMTokenList}}'s associated attribute.
+
+<p>A {{DOMTokenList}} object's
+<dfn export id=concept-domtokenlist-validation for="DOMString">validation steps</dfn> for a given <var>token</var> are:
+<ol>
```

Comment:
I would prefer some newlines here for readability. Similar to the other `<ol>`-formatted algorithms.